src/hmem_rocr.c

```diff
@@ -1054,6 +1060,48 @@ int rocr_dev_reg_copy_from_hmem(uint64_t handle, void *dest, const void *src,
 	return FI_SUCCESS;
 }
 
+bool rocr_is_dmabuf_supported(void)
+{
+	hsa_status_t hsa_ret;
+	bool dmabuf_support = false, dmabuf_export = false;
+	res = hsa_ops.hsa_system_get_info((hsa_system_info_t) 0x204, &dmabuf_support);
```

Comment:
What do you check inside `hsa_system_get_info()`?

Reply:
I'm checking whether rocr has [this field](https://github.com/RadeonOpenCompute/ROCR-Runtime/blob/master/src/inc/hsa.h#L507) initialized to be true. Edgar showed me this is also done [in RCCL](https://github.com/ROCmSoftwarePlatform/rccl/blob/develop/src/misc/rocmwrap.cc#L107C14-L107C31), i think basically if this fails there's no need to check the rest of dmabuf support conditions, but if this is enabled, it still might be a false positive